sycl/doc/extensions/experimental/sycl_ext_intel_esimd/sycl_ext_intel_esimd.md

```diff
@@ -322,13 +322,19 @@ See more details on the API documentation [page TODO](https://intel.github.io/ll
 Explicit SIMD memory access interface is quite different from the standard SYCL
 memory access interface. It supports main SYCL's device memory representations:
 - USM pointers
-- 1D global accessors
-- 2D image accessors
+- SYCL accessors
+  - 1D global accessors
+  - 1D local accessors
+  - 2D image accessors
 
-but does not `support sycl::accessor` APIs other than constructors. E.g.
-`accessor::get_pointer()`, `accessor::operator []` are not supported and must
-not be used within ESIMD code. Instead, ESIMD
-provides special APIs to access memory through these memory objects.
+Only small subset of `sycl::accessor` APIs is supported in ESIMD context:
+- accessor::accessor();
+- accessor::get_pointer(); // Supported only with `-fsycl-esimd-force-stateless-mem` switch.
+- accessor::operator[]; // Supported only with `-fsycl-esimd-force-stateless-mem` switch.
+
+ESIMD provides special APIs to access memory through accessors. Those APIs
+accept accessor object as a base reference to the addressed memory and
```

Comment:
```suggestion
an accept accessor object as a base reference to the addressed memory and
```

Reply:
you probably mean 'accept an accessor object'. I'll fix it that way.